io/disk/lvsetup.py

```diff
@@ -63,21 +63,24 @@ def setUp(self):
                 pkg = 'btrfs-progs'
         if pkg and not smm.check_installed(pkg) and not smm.install(pkg):
             self.cancel("Package %s could not be installed" % pkg)
+
```

Comment:
why this white space ?

Reply:
@PraveenPenguin Logically separating the code for installing packages (above) from the code to initiate LV related variables (below). It made the code reading easier for me.

Reply:
ok sure